pkg/collector/labels_test.go

```diff
@@ -116,3 +116,25 @@ func TestSelectorLabels(t *testing.T) {
 	// verify
 	assert.Equal(t, expected, result)
 }
+
+func TestVersionLabelImage(t *testing.T) {
+	otelcol := v1alpha1.OpenTelemetryCollector{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "my-opentelemetry-collector", 
+			Namespace: "my-namespace",
+		},
+		Spec: v1alpha1.OpenTelemetryCollectorSpec{
+			Image: "otel/opentelemetry-collector-contrib:0.56.0",
```

Comment:
Can we add another test case with a custom image?
e.g.:
```
Image: "company/opentelemetry-collector-custom:golden-stuff",
```

From my understanding we should support that too, right?